specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01-preview/botservice.json

```diff
@@ -1609,6 +1609,10 @@
           "type": "boolean",
           "description": "Whether the bot is in an isolated network"
         },
+        "disableLocalAuth": {
+          "type": "boolean",
+          "description": "Disable or enable local authentication methods." 
```

Comment:
update description to something more descriptive - Opt-out of local authentication and ensure only MSI and AAD can be used exclusively for authentication.

Reply:
Done.